content/en/building/guides/performance/telemetry.md

```diff
@@ -175,13 +175,19 @@ When the aggregate doc is created the Telemetry service also includes a snapshot
 | `dbInfo.auto_compaction` | Whether or not auto compaction is set. Added in 3.4.0. |
 | `dbInfo.adapter` | The database adapter being used. Added in 3.4.0. |
 
-## Export data to JSON
+## Export data 
+
+### Summary data via API
+
+To get summary data with one item per user, see the `/api/v2/export/user-devices` [API]({{< ref "building/reference/api#get-apiv2exportuser-devices" >}}). 
```

Comment:
I think we should mention this was added in 4.7.0

Reply:
Fair! fixed in 2a8a8669